src/main.ts

```diff
@@ -45,4 +62,112 @@ export default class DayOneImporter extends Plugin {
 	async saveSettings() {
 		await this.saveData(this.settings);
 	}
+
+	async resolveInternalLinksInNotes(): Promise<void> {
```

Comment:
Can this function be moved into a new file (e.g. `resolve-internal-links.ts`) in the same way `update-front-matter.ts` and `import-json.ts` are